app/src/utils/passwordValidations.tsx

```diff
@@ -2,7 +2,7 @@ const isPasswordTooShort = (password: string) => {
   return password.length < 8;
 };
 const doesPasswordHaveMixedCase = (password: string) => {
-  return !password.match(/[A-Z]/) || !password.match(/[a-z]/);
+  return password.match(/[A-Z]/) || password.match(/[a-z]/);
```

Comment:
Are we testing here if the password has at least one upper and one lowercase letter?
If so, I was getting unexpected results when testing. 
If you are trying to test this, this is what I came up with. There might be a more succinct way of doing it?
```suggestion
  const lowerCaseExpression = /[a-z]/;
  const upperCaseExpression = /[A-Z]/;
  return lowerCaseExpression.test(password) && upperCaseExpression.test(password);
```

Reply:
Nice catch David, I completely missed changing from or to and when I flipped the match statements. Your solution is much clearer here 🙏